.github/workflows/build_test_deploy.yml

```
@@ -527,6 +527,36 @@ jobs:
           path: |
             test/traces
 
+  testCodemods:
+    name: Test Codemods
+    runs-on: ubuntu-latest
+    needs: [build, build-native-test]
```

```suggestion
    needs: [build]
```
I don't think the codemod relies on `swc`, if not we can drop `build-native-test` from `needs`

---

Good point! I created PR #46192